src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -0,0 +1,418 @@
+        ...entityConfig,
+      }))
+      .slice(0, 3) as EntitiesCardEntityConfig[];
+
+    return html`
+      <ha-card>
+        <img src=${this.hass.hassUrl(this._config.image)} />
+        <div class="container">
+          <div class="sensors">
+            ${dialogEntities.map((entityConf) => {
+              const stateObj = this.hass!.states[entityConf.entity];
+              return html`
+                <span>
+                  <ha-icon
+                    .config=${entityConf}
+                    .icon=${stateIcon(stateObj)}
+                    .actionHandler=${actionHandler({
+                      hasHold: hasAction(entityConf.hold_action),
+                    })}
+                    @action=${this._handleAction}
+                  ></ha-icon>
+                  ${computeStateDisplay(
+                    this.hass!.localize,
+                    stateObj,
+                    this.hass!.locale
+                  )}
+                </span>
+              `;
+            })}
+          </div>
+          <div class="name">${area!.name}</div>
+          <div class="buttons">
+            ${toggleEntities.map((entityConf) => {
+              const stateObj = this.hass!.states[entityConf.entity];
+              return html`
+                <ha-icon-button
+                  .config=${entityConf}
+                  .icon=${stateIcon(stateObj)}
+                  class=${classMap({
+                    off: STATES_OFF.includes(stateObj.state),
+                  })}
+                  .actionHandler=${actionHandler({
+                    hasHold: hasAction(entityConf.hold_action),
+                  })}
+                  @action=${this._handleAction}
+                ></ha-icon-button>
+              `;
+            })}
+          </div>
+        </div>
+      </ha-card>
+    `;
+  }
+
+  protected updated(changedProps: PropertyValues): void {
+    super.updated(changedProps);
+    if (!this._config || !this.hass) {
+      return;
+    }
+    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
+    const oldConfig = changedProps.get("_config") as AreaCardConfig | undefined;
+
+    if (
+      !oldHass ||
+      !oldConfig ||
+      oldHass.themes !== this.hass.themes ||
+      oldConfig.theme !== this._config.theme
+    ) {
+      applyThemesOnElement(this, this.hass.themes, this._config.theme);
+    }
+
+    if (!this._unsubs && changedProps.has("hass")) {
+      this._loadData();
+      return;
+    }
+
+    if (!this._areas || !this._entities || !this._devices) {
+      return;
+    }
+
+    this._entitiesDialog = [];
+    this._entitiesToggle = [];
+
+    const { entities } = this._memberships(
+      this._config.area,
+      this._devices,
+      this._entities
+    );
+
+    if (entities.length === 0) {
+      return;
+    }
+
+    entities.forEach((entity) => {
+      if (!DOMAINS_TOGGLE.has(computeDomain(entity.entity_id))) {
+        this._entitiesDialog!.push(entity.entity_id);
```

Comment:
We should pick and choose what things are shown in the top right, not just all sensor types. Temperature + humidity are a good start.

Reply:
Agreed. Using sensor class is probably a good start